github_connector/views/view_github_repository.xml

```diff
@@ -45,13 +45,13 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                         <button type="action" class="oe_stat_button" icon="fa-code-fork"
                                 name="%(action_github_repository_branch)d"
                                 attrs="{'invisible': [('repository_branch_qty', '=', False)]}"
-                                context="{'search_default_repository_id': active_id}">
+                                context="{'search_default_repository_id': id}">
                             <field name="repository_branch_qty" string="Branches" widget="statinfo" />
                         </button>
                         <button type="action" class="oe_stat_button" icon="fa-users"
                                 name="%(action_github_team_repository_from_repository)d"
                                 attrs="{'invisible': [('team_qty', '=', False)]}"
-                                context="{'search_default_repository_id': active_id}">
+                                context="{'search_default_repository_id': id}">
```

Comment:
Id is working since wich version ? I allways saw only active_id.

Reply:
good question. I gave it a try and worked ;)

Reply:
#28 is related to this when you navigate for example from organization to repository and then to branch, taking incorrect `active_id`. I'm not sure if `id` is the solution to this. If not, we should change this to methods that put proper context by code using `self`.

Reply:
@pedrobaeza @legalsylvain 
unfortunately the trick doesn't work on #28 issue.
I won't have the time to work on it yet, and I would prefer to not over-extend the scope of the PR.

Would you agree ?

Reply:
OK, then please keep `active_id` for now

Reply:
@pedrobaeza I understand your concern, but I would not like to revert the change.
With `active_id` the behaviour of the instance is not correct while it is with the `id`. 
I can't see any issue with the update. Could you let me know what they are ? Then I will see if I can fix them

@legalsylvain do you see any blocking issues with `id` ?

Reply:
But I don't get what is the problem with `active_id`. Do you get some steps to reproduce the problem?

Reply:
Hello @pedrobaeza 
in our case `active_id` have the wrong behaviour.

Here is the behaviour the PR fixes:
From an organisation, go the repository using the smart button
<img width="1687" alt="odooOrga-repos" src="https://user-images.githubusercontent.com/547282/69249999-dd9b1580-0b7c-11ea-9120-6715638d3c0b.png">

Then you go to the repository branches, again using the smart button
<img width="1688" alt="account-budgeting - Odoo (20191120_1000)" src="https://user-images.githubusercontent.com/547282/69250042-f3a8d600-0b7c-11ea-9474-1bdc17dfe304.png">

From here, you will see that the filter is based on another repository, due to the fact that `active_id` refer to the `github_repository.id`. To have the proper behaviour, in the case of a deep digging using smart button, it seems it is better to use `id`. 
It is a wild case I confess.

Does it make sense ?

Reply:
Yes, but this is the same problem that it's depicted in the referred issue, and the proper solution is to switch buttons to be method based. If you are not able to do it, please remove this and I'll put a PR.